TGEngine/io/Resource.cpp

```diff
@@ -22,6 +22,7 @@ namespace tge::io {
 		// Start to read the actor
 		actorProperties.resize(actorCount);
 		actorDescriptor.resize(actorCount);
+		tge::gmc::actorInstanceDescriptor.reserve(actorCount); // Spekulative
```

Comment:
```suggestion
		tge::gmc::actorInstanceDescriptor.reserve(actorCount); // Speculative
```
actually what do you mean by speculative (as well as for any other occurance)
is it just a prediction and you don't know the exact size?

Reply:
Exactly that, because not every actor has instances. As you can see I only reserve memory and not actually resize the vector